Tests/KnitMacrosTests/SwinjectResolutionTests.swift

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright © Block, Inc. All rights reserved.
+//
+
+import Foundation
+import Swinject
+import XCTest
+import Knit
+import KnitMacros
+
+final class SwinjectResolutionTests: XCTestCase {
+    
+    func test_simple_service() {
+        let container = Factory.container
+        container.register(Service1.self, factory: Service1.make)
+        XCTAssertNotNil(container.resolve(Service1.self))
+    }
+    
+    func test_resolve_closure() {
+        let container = Factory.container
+        container.register(Service2.self, factory: Service2.make)
+        XCTAssertNotNil(container.resolve(Service2.self))
+    }
+    
+    func test_default_value() {
+        let emptyContainer = Container()
+        emptyContainer.register(Service3.self, factory: Service3.make)
+        let defaultedService = emptyContainer.resolve(Service3.self)
+        XCTAssertEqual(defaultedService?.value, 2)
+    }
+
+    /* Disabled due to import issues
```

Comment:
I can't figure out why @Argument and @Named aren't available in this file.

Reply:
What was the issue? Seems like it is fixed

Reply:
Yeah, I was having trouble but figured it out.